src/uu/ls/src/ls.rs

```diff
@@ -2416,49 +2469,32 @@ fn display_item_long(
                 ""
             },
             pad_left(&display_symlink_count(md), padding.link_count)
-        )?;
+        );
 
         if config.long.owner {
-            write!(
-                out,
-                " {}",
-                pad_right(&display_uname(md, config), padding.uname)
-            )?;
+            output_display += &format!(" {}", pad_right(&display_uname(md, config), padding.uname));
```

Comment:
We can write to strings, too: `write!(output_display, " {}", ...)`

Reply:
like this ?
`          write!(output_display," {}", pad_right(&display_uname(md, config), padding.uname))?;`

Do you know how to fix this error ?
```
error[E0277]: the trait bound `std::fmt::Error: UError` is not satisfied
    --> src/uu/ls/src/ls.rs:2474:93
     |
2474 | ...onfig), padding.uname))?;
     |                           ^ the trait `UError` is not implemented for `std::fmt::Error`
     |
```

Reply:
Oh right, writing in a string gives a fmt error. I quess we could implement that conversion. For now, I would just unwrap 😅